wiki/Disambiguation/Snapping/id.md

```diff
@@ -0,0 +1,13 @@
+# Snapping (disambiguasi)
+
+**Snapping** dapat memiliki beberapa arti:
+
+## Beatmapping
+
+- [Beat snapping](/wiki/Beatmapping/Snapping), yang merupakan posisi sebuah hit object pada sebuah ketukan musik yang telah dibagi secara otomatis oleh [beat snap divisor](/wiki/Client/Beatmap_editor/Beat_Snap_Divisor)
+- [Grid snapping](/wiki/Grid_snapping), yang merupakan posisi sebuah hit object dalam area bermain
```

Comment:
```suggestion
- [Grid snapping](/wiki/Grid_snapping), posisi dari sebuah `hit object` pada area bermain
```

Reply:
`on` than `in` is relateable here

Reply:
```suggestion
- [Grid snapping](/wiki/Grid_snapping), posisi dari sebuah *hit object* pada area bermain
```
Changed to this to make it same as above